frontend/src/components/Forms/CoffeeChatsForm/CoffeeChatsForm.tsx

```diff
@@ -30,7 +31,12 @@ const CoffeeChatsForm: React.FC = () => {
       setIsChatLoading(false);
     });
     CoffeeChatAPI.getCoffeeChatBingoBoard().then((board) => setBingoBoard(board));
-  }, [userInfo]);
+    if (member && category) {
```

Comment:
Lumping these API calls into a `useEffect` makes it so if one of them is triggered, then all calls are made. So each time I change the name in the dropdown, all three endpoints are called (instead of just `checkMemberMeetsCategory`). Maybe we can split this off into its own `useEffect` hook with the proper dependencies? And we can leave this one the way it was.